test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -35,14 +34,8 @@ describe('Snap Account - Swap', function () {
         title: this.test?.fullTitle(),
         testSpecificMock: mockSwapsTransactionQuote,
       },
-      async ({
-        driver,
-        ganacheServer,
-      }: {
-        driver: Driver;
-        ganacheServer?: Ganache;
-      }) => {
-        await loginWithBalanceValidation(driver, ganacheServer);
```

Comment:
no need to pass the localnode anymore as the default value is already '25ETH', which is what applies in the majority of the specs